### PR TITLE
fix(tools): disable i18n linting

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Disables the schema validation for non-English translation files. This will allow us to merge PRs without having to modify the i18n objects through GitHub, ensuring that we are doing those changes through Crowdin explicitly.

This comes from a discussion with @RandellDawson about how to best avoid duplicate work, and how to approach the addition of new UI elements that require translation. With this flow, a PR that adds a new UI element would only add the key to the appropriate English file. Then we would upload the change to Crowdin, pull down the translation, and merge that to land the key on the other locale files.

As it stands, there are some concerns about this approach. We'll want to establish the infrastructure to allow running CI on specific language builds, so we can adjust this to test the language file for each locale and prevent deployments with missing translation keys.

/cc @raisedadead

### CODE REVIEW

**Comment by raisedadead** [comment]
I see this discussion pop-up quite a bit. I think we should lock-in the policy. 

**TL;DR:**

It's okay if non-English sites show the exact English copy for however long it takes to get Strings translated and approved.

**The detailed version:**

Here are some of the minimum needs for deployment.

At all times there should be a commit that is deployable on all languages (zero-tolerance for variation in the commit hash being deployed). That gives us a clear path to deployments for emergency patches (especially security & show stoppers).

A commit's "deployability" is determined with linting checks & CI checks.

I agree this means we would be needing to make a trade-off somewhere. Here is a workflow that we should enforce that establishes that tradeoff:

<details>
<summary>Workflow diagram</summary>

translation-flow

</details>

This means with PRs like #41002 we will overwrite the changes on the repo. Those could either be a manual on-off translation to non-English files we make over in #41002 or an English string that was copy pasted over to the non-English file.

One other thing, we stop using these debug bits like "ES Translate help" etc. When need of test bits let's just use Google translate. I understand it's not accurate - and that is the point to get a true copy from Crowdin via the policy above. You can think of it as getting an improved translation of original string.

I intentionally used a Google translated bit in the screenshots on #41002 to see how it fits.

So, there it's **always nice to** use a Google translated version for testing and debugging, and when making a PR copy paste over the exact English string to non-English counterpart.

That way we can keep our sanity with the linting and testing we have in place for deployments. 

@nhcarrigan, 

You should adjust this PR if needed accordingly.

**Comment by nhcarrigan** [comment]
> when making a PR copy paste over the exact English string to non-English counterpart.

With the policy being that we copy the English key-value pairs to the non-English files, this PR can be closed and we can retain the same lint step we currently have, which asserts that all translation objects match the correct schema.

**Comment by raisedadead** [comment]
Perfect, in that case we should just update the PR to add some docs instead for reference.

**Comment by RandellDawson** [comment]
Can we discuss this further?  I don't like the idea of having English scattered throughout the home page of the Chinese site.  It is one thing to do this for a section of the site (where we have a note about needing help for translation), but for the main part of the site to have a mix of English and non-English seems strange to me.

**Comment by ojeytonwilliams** [comment]
I think the key point is we need an 'emergency path to deployment' and that's the only time we *might* have English.

**Comment by nhcarrigan** [comment]
Going to close this, since we are implementing the flow of adding the English key-value pairs to the i18n objects.

Will create a separate PR for the documentation discussed above.
